useIsShake.js

```diff
@@ -69,15 +50,13 @@ export function useIsShake() {
   };
 
   useEffect(() => {
-    _subscribe();
-
-    const onLoad = setTimeout(() => {
-      isToggleReady = true;
-      clearTimeout(onLoad);
-    }, SVG_LOAD_DURATION);
-
+    // invocation of _subscribe when isToggleReady === true
+    if (isShakeReady) {
+      _subscribe();
+      // isShake Accelerometer listener is removed when !isToggleReady
+    } else _unsubscribe();
     return () => _unsubscribe();
-  }, []);
+  }, [isShakeReady]);
 
-  return { toggle };
+  return { isShakeTriggered, setIsShakeReady };
```

Comment:
`toggle` renamed to `isShakeTriggered` and `setIsShakeReady` being returned here to give animation wrapper components ability to affect debouncing of shake triggering.

Reply:
remove, this is clear from code change